wallet-services/src/main/java/org/exoplatform/wallet/service/WalletAccountServiceImpl.java

```diff
@@ -170,19 +169,18 @@ public void refreshWalletFromBlockchain(Wallet wallet,
       LOG.warn("Can't refresh wallet from blockchain because TokenAdminService isn't initialized yet");
     } else {
       try {
-        Set<String> walletModifications = walletsModifications == null ? null
-                                                                       : walletsModifications.get(wallet.getAddress());
+        Set<String> walletModifications = walletsModifications == null ? null : walletsModifications.get(wallet.getAddress());
         accountStorage.retrieveWalletBlockchainState(wallet, contractDetail.getAddress());
-        getTokenAdminService().retrieveWalletInformationFromBlockchain(wallet,
-                                                                       contractDetail,
-                                                                       walletModifications);
+        getTokenAdminService().retrieveWalletInformationFromBlockchain(wallet, contractDetail, walletModifications);
         saveWalletBlockchainState(wallet, contractDetail.getAddress());
 
-        if (!StringUtils.equalsIgnoreCase(wallet.getInitializationState(), WalletState.INITIALIZED.name()) &&
-            !StringUtils.equalsIgnoreCase(wallet.getInitializationState(), WalletState.PENDING.name()) &&
-            !StringUtils.equalsIgnoreCase(wallet.getInitializationState(), WalletState.DENIED.name())
+        if (!StringUtils.equalsIgnoreCase(wallet.getInitializationState(), WalletState.INITIALIZED.name())
+            && !StringUtils.equalsIgnoreCase(wallet.getInitializationState(), WalletState.PENDING.name())
```

Comment:
please avoid formatting the whole code and add in PR the modified code only

Reply:
done !